carma_cloud_client/README.md

```diff
@@ -0,0 +1,3 @@
+# carma_cloud_client
+
+TODO for USER: Add description of package and link to confluence documentation.
```

Comment:
Maybe create a brief detailed design for this carma_cloud_client in confluence and add the confluence  link here

Reply:
added